lib/widgets/twake_components/twake_preview_link.dart

```diff
@@ -31,6 +32,10 @@ class TwakePreviewLink extends StatelessWidget {
         removeElevation: true,
         bodyMaxLines: 4,
         borderRadius: 12,
+        placeholderWidget: TwakePreviewPlaceHolder(
+          width: MediaQuery.of(context).size.width,

```

Comment:
Is this supporting web ?

Reply:
any_link_preview lib doesn't seem to work in web version. link doesn't show in main branch aswell 
![image](https://github.com/linagora/twake-on-matrix/assets/48354990/758d8d77-4ea3-4d8c-81b7-922a71ed07d5)